tools/depends/target/boost/CMakeLists.txt

```diff
@@ -0,0 +1,129 @@
+cmake_minimum_required(VERSION 3.16)
```

Comment:
Should you maybe look at leveraging https://github.com/boostorg/cmake rather than a rag tag custom cmake that potentially becomes a maintenance burden long term.

Reply:
Yes, this resource wasn't available 4 years ago when I first ported boost for IPFS/libp2p.